doc/build-unix.md

```diff
@@ -222,3 +222,21 @@ In this case there is no dependency on Berkeley DB 4.8.
 Mining is also possible in disable-wallet mode, but only using the `getblocktemplate` RPC
 call, not `getwork`.
 
+Snapcraft
+====
+
+You can also build Dogecoin Core with [Snapcraft](https://snapcraft.io/) on supported systems by pulling the source repo, and running 'snapcraft' within the root of the master branch.
+
+The resulting .snap file can then be installed locally via:
+
+    sudo snap install --dangerous <filename>
```

Comment:
Just the fact that it's being built locally, and so isn't coming from a trusted (cryptographically-signed) source. If there was interest in maintaining a compiled snap on the Snap Store, then this becomes a lot simpler.